### PR TITLE
Ports Pixel shifting.

### PR DESCRIPTION
**Pull request by Detective-Google**
you can now shift your character over in their tile by holding SHIFT+CTRL and pressing an arrow key. 
https://imgur.com/a/j71gD1g

resets you to 0 when you move for convenience & to prevent abuse.
Limited to 16 pixels in any direction

### CODE REVIEW

**Comment by KasparoVy** [comment]
Such an awesome feature.

**Comment by ariever** [comment]
Now i can cuddle

**Comment by Iroquois-Pliskin** [comment]
Donuddd...
Yes please, cool and good feature for when I want to RP absolutely staring someone to death.

**Comment by chaoko99** [comment]
Polaris will **never** merge this Verk, they have no reason for it.

File: code/modules/mob/mob_defines.dm [hunk 1]
```diff
@@ -83,6 +83,7 @@
 	var/resting = 0			//Carbon
 	var/lying = 0
 	var/lying_prev = 0
+	var/is_shifted = FALSE // Vore Edit; pixel shifting
```
Comment (Novacat) [inline code/modules/mob/mob_defines.dm:86]: This should have VOREStation Edit, mostly to make it easier to find edits via the search feature.

**Pull request merged**